source/features/last-notification-page-button.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import looseParseInt from '../helpers/loose-parse-int';
+import {assertNodeContent} from '../helpers/dom-utils';
+
+function init(): void | false {
+	const nextButton = select('a[data-hotkey="ArrowRight"]')?.cloneNode();
+	if (!nextButton) {
+		return false;
+	}
+
+	const lastNotificationPageNode = select('.js-notifications-list-paginator-counts')!.lastChild!;
+	assertNodeContent(lastNotificationPageNode, ' of ');
```

Comment:
Assertions are done after trimming

```suggestions
	assertNodeContent(lastNotificationPageNode, 'of');
```